wallet/models.py

```diff
@@ -21,8 +24,8 @@ def generate_wallet_name():
             return generate_wallet_name()  # noqa F821
 
     name = models.CharField(
-        max_length=8, unique=True, default=generate_wallet_name
-    )  # noqa E501
+        max_length=8, unique=True, default=generate_wallet_name, editable=False
+    )
     type = models.CharField(choices=WALLET_TYPE, max_length=100)
     currency = models.CharField(choices=CURRENCY, max_length=3)
```

Comment:
please, set default values for `type` and `currency` fields
e.g. type=visa, currency=euro

Reply:
Done